transforms/angle-brackets/transform.js

```diff
@@ -280,6 +280,16 @@ function hasValuelessDataParams(params) {
   return getDataAttributesFromParams(params).length > 0;
 }
 
+function shouldSkipDataTestDataParams(node, includeValuelessDataTestAttributes) {
+  // data-* attributes are generally omitted,
+  // but this config allows including data-test-* attributes.
+  if (includeValuelessDataTestAttributes) {
+    const dataAttrs = getDataAttributesFromParams(node.params);
+    return !dataAttrs.some(p => p.original.indexOf('data-test') === 0);
```

Comment:
nothing wrong with the above statement but may be `startsWith` would be a better choice here

Reply:
Thanks